modules/docs/sc_param.md

```diff
@@ -69,7 +69,7 @@ The sc_param module provides methods to help you handle parameters for your stre
-| max_all_queues_age                      | number | 300                                                                           | if the last global flush date was 300 seconds ago, it will force a flush of each queue                                                                                                                                                                                                                                                            |                                                                                                                                                                                                                                                                                          |                                                                                                                                                                                                                                                                                                  |
+| max_all_queues_age                      | number | 5                                                                           | if the last global flush date was 5 seconds ago, it will force a flush of each queue                                                                                                                                                                                                                                                            |                                                                                                                                                                                                                                                                                          |                                                                                                                                                                                                                                                                                                  |
```

Comment:
```suggestion
| max_all_queues_age                      | number | 5                                                                           | if the last global flush was 5 seconds ago, it will force a flush of each queue                                                                                                                                                                                                                                                            |                                                                                                                                                                                                                                                                                          |                                                                                                                                                                                                                                                                                                  |
```